core/functional_tests/websocket/service.cpp

```diff
@@ -90,10 +90,37 @@ class WebsocketsFullDuplexHandler final
   }
 };
 
+class WebsocketsFullDuplexHandlerAlt final
```

Comment:
It does not look like a full-duplex. The data is not send and received concurrently from different threads/coroutines. Better name it `WebsocketsHandlerAlt`